addons/web/static/src/scss/base_frontend.scss

```diff
@@ -15,7 +15,10 @@ html, body, #wrapwrap {
     // This is at least needed for the edit mode to not have a double scrollbar
     // due to the right editor panel (and since we want to minimize the style
     // difference between edit mode and non-edit mode (wysiwyg)...).
-    overflow: auto;
+    overflow-y: auto;
+    // Because of website animate, some elements overflow to the right of the
+    // wrapwrap while waiting to be animated. So we have to remove the horizontal scrollbar.
+    overflow-x: hidden;
```

Comment:
I think we already talked about it but can't remember the conclusion. I still think this is not good: you don't want to hide all overflowing contents. The user could have put a really long link or a table or anything else that you may want to be able to see on all device sizes. This would also hide bugs where we make something in a feature overflow by mistake.

What about:
1. Not placing the animated elements outside of the viewport while they are not being animated yet (they can simply be `visibibility: hidden` during that time)
2. Set `overflow-x: hidden` like you did here... but only for the duration of the animation.

Reply:
ok, du coup le commit restant ici n'est plus necessaire que si vraiment on a une animation qui se declenche au moment ou on scrolle la page en draggant un snippet